test/unified_malloc_framework/memoryPoolAPI.cpp

```diff
@@ -157,6 +157,18 @@ INSTANTIATE_TEST_SUITE_P(
             .second;
     }));
 
+INSTANTIATE_TEST_SUITE_P(
+    proxyPoolOOMTest, umfMemTest,
+    ::testing::Values(std::tuple(
+        [] {
+            return umf::poolMakeUnique<umf_test::proxy_pool, 1>(
+                       {umf::memoryProviderMakeUnique<
+                            umf_test::provider_mock_out_of_mem>(10)
+                            .second})
+                .second;
+        },
+        0)));
```

Comment:
You set the provider to be OOM after 10 allocations but at the same time set the `expectedPooledAllocations` as 0.

Reply:
Yes, that's because there's no pooling for proxy pool so the part of the test where the `expectedPooledAllocations` is used, for now is only done for disjoint pool.

Reply:
Oh, my bad.